panda/src/cocoadisplay/CMakeLists.txt

```diff
@@ -31,5 +33,3 @@ find_library(APPKIT_LIBRARY AppKit)
 find_library(CARBON_LIBRARY Carbon)
 target_link_libraries(p3cocoadisplay
   ${APPLICATIONSERVICES_LIBRARY} ${APPKIT_LIBRARY} ${CARBON_LIBRARY})
-
-install(TARGETS p3cocoadisplay DESTINATION lib)
```

Comment:
Shouldn't the p3cocoadisplay dylib be installed so that libpandagl can link against it?